jmespath/functions.py

```diff
@@ -1,5 +1,6 @@
 import math
 import json
+from typing import Sequence
```

Comment:
this is deprecated and should be
```suggestion
from collections.abc import Sequence
```
I think

Reply:
That's only available in Python 3.9+ but the project still declares support for 3.7 and 3.8

Reply:
it's available in older versions. only subscripting like `Sequence[int]` (which is not used) is only available in newer versions.